wallet/src/components/Proposal.tsx

```diff
@@ -259,8 +259,14 @@ const Proposal = ({ offer, purses, swingsetParams, beansOwing }) => {
       {offerArgs !== undefined && (
         <div className="OfferEntry">
           <h6>Arguments</h6>
-          <details style={{ whiteSpace: 'pre-wrap' }}>
-            {stringify(offerArgs, true)}
+          <details
+            style={{
+              whiteSpace: 'pre-wrap',
+            }}
+          >
+            <Box sx={{ fontFamily: '"RobotoMono", monospace' }}>
```

Comment:
I'm surprised that `pre-wrap` doesn't use `<pre>` style. 

WDYT of using `<pre>` instead of htis fontFamily declaration?

Reply:
`<pre>` doesn't seem to format the same way, some of the text overflows